bin/impl/base.sh

```diff
@@ -38,5 +38,5 @@ export WI_DATA_DEP_JAR=$WI_HOME/modules/data/target/webindex-data-0.0.1-SNAPSHOT
 if [ ! -f $WI_DATA_DEP_JAR ]; then
   echo "Building $WI_DATA_DEP_JAR"
   cd $WI_HOME
-  mvn clean package -DskipTests
+  mvn clean package -DskipTests -Dspark.version=$SPARK_VERSION -Dhadoop.version=$HADOOP_VERSION
```

Comment:
Could have a sanity check to ensure these are set.  Like this script checks that some other vars are set.